proto/http.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package edgehog_device_forwarder_proto;
+
+message Http {
+    oneof message {
+        Request request = 1;
+        Response response = 2;
+    }
+
+    message Request {
```

Comment:
We also need a `port` here, otherwise we can't send a request to a specific device-local port (e.g. a process running on `localhost:8080`)

Reply:
Done. I used an uint32 for the port type that will be converted into u16 in the Rust code.